repos/system_upgrade/common/actors/setuptargetrepos/tests/test_setuptargetrepos.py

```diff
@@ -205,14 +212,14 @@ def test_repos_mapping_for_distro(monkeypatch, distro_id):
     produced_rhel_repoids = {repo.repoid for repo in rhel_repos}
 
     expected_repoids = {
-        "{0}-9-for-x86_64-baseos-htb-rpms".format(distro_id),
-        "{0}-9-for-x86_64-appstream-htb-rpms".format(distro_id),
-        "{0}-9-for-x86_64-satellite-extras-rpms".format(distro_id),
+        "distro-9-for-x86_64-baseos-htb-rpms",
+        "distro-9-for-x86_64-appstream-htb-rpms",
+        "distro-9-for-x86_64-satellite-extras-rpms",
     }
 
     assert produced_distro_repoids == expected_repoids
-    if distro_id == 'rhel':
+    if dst_distro == 'rhel':
         assert len(rhel_repos) == 3
         assert produced_rhel_repoids == expected_repoids
     else:
-        assert len(rhel_repos) == 0
+        assert rhel_repos == []  # better diff than len comparison
```

Comment:
that could be already then

```suggestion
        assert not rhel_repos
```

no need to change it.